docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -151,7 +148,7 @@ auto --batch \
 -Expack.security.http.ssl.certificate=certificates/es01/es01.crt \
 -Expack.security.http.ssl.certificate_authorities=certificates/ca/ca.crt \
 -Expack.security.http.ssl.key=certificates/es01/es01.key \
---url https://localhost:9200"
+--url https://es01:9200"
```

Comment:
I think it would be better to avoid localhost entirely. IT would be good to be able to get rid of `localhost` and `127.0.0.1` in the `instances.yml` too but thats not possible right now